### ISSUE TITLE
coho audit-license-headers issues in cordova-fetch

### ISSUE THREAD

**Issue by brodybits** [apache/cordova-fetch#35]
_Migrated_ from <https://issues.apache.org/jira/browse/CB-14256>:

`coho audit-license-headers -r fetch` uncovered the following text files without a valid license header:

- `spec/support/dummy-local-plugin/plugin.xml`
- `spec/support/repo-name-neq-plugin-id.git/HEAD`
- `spec/support/repo-name-neq-plugin-id.git/config`
- `spec/support/repo-name-neq-plugin-id.git/refs/heads/master`

On the `1.3.x` branch this issue only shows up in `spec/support/dummy-local-plugin/plugin.xml` at this time.

Recommended solution is to add standard header to `spec/support/dummy-local-plugin/plugin.xml` and add the other files to `.ratignore`.

**Comment by raphinesse** [comment]
I always wondered what these .ratignore files were doing. A quick search only told me that it was used by some Apache software. Could you enlighten me @brodybits? So that hopefully, next time I know which new files to add to it.

**Comment by brodybits** [comment]
One of the steps to making a release according to cordova-coho docs is to do `coho audit-license-headers`, which uses the Apache "rat" tool to check that all text files have an appropriate license header. I discovered the ` .ratignore` file can be used to ignore certain files, which seems to be mostly used for JSON files but I think should also be used for ".git" artifacts like were added in 36ecb8c105dd6b2bc0c18f9abd30cc444150e8db (GH-24). It seems like this step is normally taken during the release process.
